src/index.html

```diff
@@ -5,10 +5,29 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <div class="icon">
+        <a href="#head"><img src="./images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item is-active"><a href="#">apple</a></li>
```

Comment:
it would be great to add more meaningful links
```suggestion
          <li class="nav__item is-active"><a href="#apple">apple</a></li>
```